stat/stat.go

```diff
@@ -1293,3 +1294,76 @@ func MeanVariance(x, weights []float64) (mean, variance float64) {
 	variance = (ss - compensation*compensation/sumWeights) / (sumWeights - 1)
 	return mean, variance
 }
+
+// NormalSkewTest test whether the skew is different from the normal distribution.
+// This function tests the null hypothesis that the skewness of population that the
+// sample was drawn from is the same as that of a corresponding normal distribution.
+// Implementation based on scipy's: https://github.com/scipy/scipy/blob/v1.3.0/scipy/stats/stats.py#L1481-L1549
+func NormalSkewTest(a []float64) float64 {
+	b2 := Skew(a, nil)
+	n := float64(len(a))
+	if n < 8 {
+		panic(fmt.Sprintf("skewtest is not valid with less than 8 samples, %d samples were given", int(n)))
+	}
+	y := b2 * math.Sqrt(((n+1)*(n+3))/(6.0*(n-2)))
```

Comment:
The `X.0` syntax is not required since the expression is in a float context, throughout.